### ISSUE TITLE
Warning doesn't make sense

### ISSUE THREAD

**Issue by ionelmc** [timothycrosley/isort#1367]
```
/home/ionel/.cache/pre-commit/repo4uc6wwmz/py_env-python3.8/lib/python3.8/site-packages/isort/settings.py:312: UserWarning: `known_thirdparty` setting is defined, but not THIRDPARTY is not included in `sections` config option: ('FUTURE', 'STDLIB', 'THIRDPARTY', 'FIRSTPARTY', 'LOCALFOLDER').
  warn(
```

I don't get what's with the double negation over there. Can this be rephrased in plain English?

**Comment by timothycrosley** [comment]
I agree the warning makes no sense! I'll work on fixing the wording before the next release, in the meantime the issue is that the setting is `known_third_party` not `known_thirdparty` as noted in the documentation here: https://github.com/timothycrosley/isort/#custom-sections-and-ordering. `known_thirdparty` has never been a setting, so if categorization is working as intended, likely it just needs to be removed from your config.
